docs/MO_DG/prepare_model/Supported_Frameworks_Layers.md

```diff
@@ -259,7 +327,10 @@ Standard TensorFlow\* operations:
 | Tile | No |
 | TopkV2 | No |
 | Transpose | No |
+| Unique | No |
```

Comment:
This operation is actually not supported. MO generates it but nGraph does not support